### PR TITLE
RHIROS-170 Add RHEL8 for ARM arch

### PR DESCRIPTION
**Pull request by apuntamb**
arm_ansible

### CODE REVIEW

File: ansible-playbooks/ros_install_and_set_up.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@
   tasks:
     - name: Enable required repository
       rhsm_repository:
-        name: rhel-8-for-x86_64-baseos-rpms,rhel-8-for-x86_64-appstream-rpms
+        name: rhel-8-for-x86_64-baseos-rpms,rhel-8-for-x86_64-appstream-rpms,rhel-8-for-aarch64-baseos-rpms,rhel-8-for-aarch64-appstream-rpms
```
Comment (kgaikwad) [inline ansible-playbooks/ros_install_and_set_up.yml:12]: Will it be possible to make use of ansible facts?

Comment (apuntamb) [reply ansible-playbooks/ros_install_and_set_up.yml]: done

Comment (apuntamb) [reply ansible-playbooks/ros_install_and_set_up.yml]: Tested on ARM. Works as expected.

**Review by kgaikwad — CHANGES_REQUESTED** [review]

File: ansible-playbooks/ros_install_and_set_up.yml (hunk 1 above)
Comment (kgaikwad) [reply ansible-playbooks/ros_install_and_set_up.yml]: Could you please also attach some logs for reference?

Comment (apuntamb) [reply ansible-playbooks/ros_install_and_set_up.yml]: Attached screenshot at top.

File: ansible-playbooks/ros_install_and_set_up.yml [hunk 2]
```diff
@@ -6,11 +6,19 @@
 # 4. Start and enable pmlogger service
 
 - hosts: all
+  gather_facts: True
   tasks:
-    - name: Enable required repository
+    - debug: var=ansible_architecture
+    - name: Enable required repository for x86_64 arch
+      when: ansible_architecture == "x86_64"
```
[the remark this answers is not in the document]
Comment (apuntamb) [reply ansible-playbooks/ros_install_and_set_up.yml]: done

File: ansible-playbooks/ros_install_and_set_up.yml [hunk 3]
```diff
@@ -6,11 +6,19 @@
...
       rhsm_repository:
         name: rhel-8-for-x86_64-baseos-rpms,rhel-8-for-x86_64-appstream-rpms
         state: enabled
+    - name: Enable required repository for ARM arch
+      when: ansible_architecture == "aarch64"
```
[the remark this answers is not in the document]
Comment (apuntamb) [reply ansible-playbooks/ros_install_and_set_up.yml]: done

**Review by kgaikwad — APPROVED** [review]
LGTM :+1:

**Pull request merged**